src/components/Table/Table.vue

```diff
@@ -39,11 +51,15 @@
 export interface Field {
     key: string;
     label?: string;
+    sortable?: boolean;
+    asc?: boolean;
+    desc?: boolean;
```

Comment:
Would this not be a better solution?
```suggestion
    sortable?: boolean;
    sort?: 'asc' | 'desc';
```
Because does the following even make sence?
```ts
{
    asc: true,
    desc: true,
}
```